### PR TITLE
Relocate contributing guidelines

### LINKED ISSUE #2150: Relocate info from readme to contributing section
Our README.md contains quite a bit of information about contributing to the documentation and running it as a project.

As mentioned in [this issue](https://github.com/meilisearch/documentation/issues/1881) we should relocate this information into the [contribution section of the documentation website](https://docs.meilisearch.com/learn/contributing/overview.html) and refocus the content of `README.md` around the Meilisearch project and company.

README could contain:
- A header that's similar to those found on other projects in the Meili org, w/ links to other Meilisearch repos and tools 
- A short description of the project's purpose: documentation for the Meilisearch search engine API
- A link to the documentation website
- Possibly, some aspects of contribution that make sense to keep on GitHub, e.g. the handles of the project's maintainers

### PR DESCRIPTION
**Pull request by james-2001**
# Pull Request
Moved information about contributing to the documentation from README.markdown into learn/contributing.

## Related issue
Fixes #2150 
See also #1881

## What does this PR do?
- Moves information about the contributing guidelines from the readme into files in learn/contributing. This allows them to be rendered on the website
-  Adds useful links to the README of the repository.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: learn/contributing/contributing_to_docs.md [hunk 1]
```diff
@@ -91,3 +93,83 @@ We ask the following questions:
...
+
+```bash
+yarn style:fix
+```
+
+You can test if the code is well-formatted without fixing it.
+
+```bash
+yarn style
+```
```
Comment (maryamsulemani97) [inline learn/contributing/contributing_to_docs.md:156]:
```suggestion
Run the following command to check for any styling errors like extra spaces or lines in `*.vue`, `*.js`, and `*.md` files:

```bash
yarn style
```

You can fix the styling errors using:

```bash
yarn style:fix
```
```

File: learn/contributing/contributing_to_docs.md [hunk 2]
```diff
@@ -91,3 +93,83 @@ We ask the following questions:
...
+
+### Handling images and other assets
+
+Screenshots, images, GIFs, and video demonstrations should be placed in a relevant folder under `.vuepress/public/` and then referenced from any markdown file a using relative link. For example, if you create the file `.vuepress/public/my_cool_guide/my_cool_image.png`, you would link to it from your document using
```
Comment (maryamsulemani97) [inline learn/contributing/contributing_to_docs.md:160]:
```suggestion
Screenshots, images, GIFs, and video demonstrations should be placed in a relevant folder under `.vuepress/public/` and then referenced from any markdown file a using relative link. For example, if you create the file `.vuepress/public/my_cool_guide/my_cool_image.png`, you would link to it to your document using:
```

File: README.markdown [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Meilisearch Documentation
```
Comment (maryamsulemani97) [inline README.markdown:1]:
```suggestion
<p align="center">
  <img src="assets/meilisearch-logo-light.svg?sanitize=true#gh-light-mode-only">
  <img src="assets/meilisearch-logo-dark.svg?sanitize=true#gh-dark-mode-only">
</p>

<h1 align="center">Meilisearch Documentation</h1>
```

You cam add the logos from [here](https://github.com/meilisearch/meilisearch/tree/main/assets)

**Review by maryamsulemani97 — CHANGES_REQUESTED** [review]
Hey @james-2001! 👋 
Thank you for your PR! I made some suggestions. Please let me know if you have any questions!

File: learn/contributing/contributing_to_docs.md (hunk 2 above)
Comment (james-2001) [reply learn/contributing/contributing_to_docs.md:161]: Possibly a better wording would be "you would **embed it** in your document using:", neither of these two phrasings above sound particularly good to me. Thoughts?

File: learn/contributing/contributing_to_docs.md [hunk 4]
```diff
@@ -91,3 +93,75 @@ We ask the following questions:
...
+
+```bash
+yarn check-links
+```
+
+#### Checking styling
+
+Run the following command to check for any styling errors like extra spaces or lines in `*.vue`, `*.js`, and `*.md` files:
+
+```bash
```
Comment (maryamsulemani97) [inline learn/contributing/contributing_to_docs.md:147]: GitHub is messing up the formatting when I try suggesting changes. Can you please fix the formatting for lines 147-149. It should be one code block wrapped in ```

File: learn/contributing/contributing_to_docs.md [hunk 5]
```diff
@@ -91,3 +93,75 @@ We ask the following questions:
...
+yarn style
+
+### Handling images and other assets
+
+Screenshots, images, GIFs, and video demonstrations should be placed in a relevant folder under `.vuepress/public/` and then referenced from any markdown file a using relative link. For example, if you create the file `.vuepress/public/my_cool_guide/my_cool_image.png`, you would link to it from your document using 
+
+```md
```
Comment (maryamsulemani97) [inline learn/contributing/contributing_to_docs.md:154]:
```suggestion
```
```

**Review by maryamsulemani97 — APPROVED** [review]
LGTM! Thank you for the PR @james-2001! 🙏 
I added the assets folder before merging this.

bors merge

**Pull request merged**